package NYT.NNodeTrackerServer.NProto;

import "yt/ytlib/node_tracker_client/node.proto";
import "yt/ytlib/node_tracker_client/node_tracker_service.proto";

////////////////////////////////////////////////////////////////////////////////

message TReqRegisterNode
{
    required NYT.NNodeTrackerClient.NProto.TNodeDescriptor node_descriptor = 1;
    required NYT.NNodeTrackerClient.NProto.TNodeStatistics statistics = 2;
}

message TRspRegisterNode
{
    required int32 node_id = 1;
}

message TReqUnregisterNode
{
    required int32 node_id = 1;
}

////////////////////////////////////////////////////////////////////////////////
